app/modules/wallet/addresses/addresses.controller.ts

```diff
@@ -0,0 +1,56 @@
+import * as restify from 'restify';
+import { InternalServerError, NotFoundError } from 'restify-errors';
+import IController from '../../../interfaces/utils/IController';
+import IRequest from '../../../interfaces/utils/IServer';
+import WalletAddress from '../../../models/wallet_address.model';
+import { IWallet } from '../../../interfaces/models';
+
+export default class WalletsController implements IController {
```

Comment:
You named it `WalletsController` however, the filename depicts that its `WalletsAddressesController`?